wallet/wallet-cli-lib/tests/cli_test_framework.rs

```diff
@@ -16,11 +16,14 @@
 use blockprod::{rpc::BlockProductionRpcServer, test_blockprod_config};
 use crypto::{key::PublicKey, random::Rng, vrf::VRFPublicKey};
 use hex::FromHex;
+use tokio::task::JoinHandle;
 
 use std::{
-    collections::VecDeque,
     net::SocketAddr,
-    sync::{Arc, Mutex},
+    sync::{
+        mpsc::{Receiver, Sender},
```

Comment:
Minor: Maybe import just `mpsc` and refer to the channel endpoint types as `mpsc::Sender` and `mpsc::Receiver` in the code. It's a bit clearer since the other kinds of channels also have a `Sender` and a `Receiver`.